addons/mail/models/mail_message.py

```diff
@@ -1025,6 +1016,20 @@ def _message_format(self, format_reply=True, msg_vals=None, for_current_user=Fal
                 vals["trackingValues"] = displayed_tracking_ids._tracking_value_format()
         return vals_list
 
+    def _reaction_groups(self):
```

Comment:
I'm wondering whether we should keep this in message or move it to reaction model.
If kept here, it should probably be after `_message_format_extras`.

Reply:
I think the best would be to move to reaction, eventually I'll convert it `_to_store`.

Reply:
Moved to `mail.message.reaction`